link `at` and `with` together (#29099)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/at/index.md

The usual practice is to access {{jsxref("Array/length", "length")}} and calculate the index from that — for example, `array[array.length - 1]`. The `at()` method allows relative indexing, so this can be shortened to `array.at(-1)`.

> By combining this with {{jsxref("Array.prototype.with()", "with()")}}, you can both read and write (respectively) the array with negative indices.

Reviewer: ```suggestion
By combining `at()` with {{jsxref("Array/with", "with()")}}, you can both read and write (respectively) an array using negative indices.
```
